packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -476,4 +476,16 @@
       <label>Copié!</label>
       <name>quantic_Copied</name>
     </customLabels>
+    <customLabels>
```

Comment:
@acarpentier01 the translations are here as well, all your suggestions are very welcome!